### ISSUE TITLE
[Request]: Add option to spawn inside the vehicle to the "Persistent" and "Xml" parts in Vehicle spawner

### ISSUE THREAD

**Issue by gamerboy1992** [Yimura/YimMenu#1657]
### Solution

Have the option "Spawn Inside" or "Spawn In" added to the "Persistent" and "Xml" tab in "Spawn Vehicle".

### Reason

I realised that I can only spawn inside my json or xml vehicle if I have the option "Spawn Inside" enabled in the "New" part of the vehicle spawner. That can get a bit annoying sometimes but it's a very minor detail.

**Comment by Tayipc38** [comment]
That is a good point. Would it also bother you if enabling one of them enabled all of them?

**Comment by gamerboy1992** [comment]
I don't think it would be a bother if enabling one would enable all. I just mentioned this because there is individual option for the "New" and "Personal" but the option for "New" also applies to the "Persistent" and "Xml" which makes it a bit inconsistent.

**Comment by Tayipc38** [comment]
> I don't think it would be a bother if enabling one would enable all. I just mentioned this because there is individual option for the "New" and "Personal" but the option for "New" also applies to the "Persistent" and "Xml" which makes it a bit inconsistent.

You are definitely right about it being inconsistent. But I was wondering if having separate globals for each would be desirable.

**Comment by gamerboy1992** [comment]
Then I guess putting the option in each tab is the way that doesn't have such undesirable effects whilst keeping it consistent.
